modules/bibsort/etc/bibsort.cfg

```diff
@@ -17,25 +17,30 @@
 
 [sort_field_1]
 name = latest first
-definition = BIBREC: creation_date
+definition = BIBREC: id
 washer = NOOP
```

Comment:
Correct. Indeed mine was only a micro-optimization that we can very well skip (especially in the situation you describe and also in the context of the enforced 005).